syngenta_digital_dta/mongo/adapter.py

```diff
@@ -46,6 +46,16 @@ def create(self, **kwargs):
         super().publish('create', data, **kwargs)
         return data
 
+    def batch_create(self, **kwargs):
+        items = []
+        for item in kwargs['data']:
+            item = schema_mapper.map_to_schema(item, self.model_schema_file, self.model_schema)
+            item['_id'] = item[self.model_identifier]
+            items.append(item)
+        self.connection.insert_many(items)
+        super().publish('create', items, **kwargs)
```

Comment:
Downstream users will be need to know that this is sending a list of items

```suggestion
        super().publish('batch_create', items, **kwargs)
```

Reply:
Sure